client/packages/system/src/Patient/ListView/ListView.tsx

```diff
@@ -11,8 +11,8 @@ import {
   ColumnAlign,
   useAlertModal,
   useTranslation,
-  useUrlQueryParams,
   ReadOnlyCheckboxCell,
+  useQueryParamsStore,
```

Comment:
I also don't think that this page should be changed. It makes the list view different from all others in the site and removes the ability to bookmark the list view sort options

Reply:
thanks!, yes I was a bit eager I used the url parameter again for the patient end encounter list view.

How the API hooks are used here is a bit confusing too me. For example, the "list encounter" hook was reused from the "main" encounter list view which uses a different hook to obtain the query params, I needed a while to track this down and fix it. Maybe something to discuss at a later point.